Sources/SwiftFormat/PrettyPrint/Token.swift

```diff
@@ -196,6 +198,13 @@ enum Token {
   /// Ends a scope where `contextual` breaks have consistent behavior.
   case contextualBreakingEnd
 
+  /// Turn formatting back on at the given position in the original file
+  /// nil is used to indicate the rest of the file should be output
+  case enableFormatting(AbsolutePosition?)
+
+  /// Turn formatting off at the given position in the original file.
+  case disableFormatting(AbsolutePosition)
+
```

Comment:
Maybe I’m missing something here but why do `enableFormatting` and `disableFormatting` need to carry a position? Shouldn’t we enable/disable formatting exactly where the `enableFormatting` and `disableFormatting` token occur within the token stream?

---

Design question for @allevato: Should these be unified into the `printControl` case? Or would it make sense to have a single case `enableFormattingControl` with a sub-enum like `PrinterControlKind` as we do for `printerControl`?

Reply:
> Design question
I'd certainly prefer that these stand alone. They don't seem like "sub-enums" to me.